controllers/storagecluster/cephcluster.go

```diff
@@ -1032,7 +1034,7 @@ func generateMgrSpec(sc *ocsv1.StorageCluster) rookCephv1.MgrSpec {
 	}
 	if sc.Spec.Mgr != nil && sc.Spec.Mgr.EnableActivePassive {
 		spec.Count = 2
```

Comment:
What about setting the mgr count to 1 if it's single node?

Reply:
In a single node setup, we wouldn't enable active/passive mgr right? So, count would still be 1.